packages/client/tests/functional/order-by-null/tests.ts

```diff
@@ -0,0 +1,160 @@
+import testMatrix from './_matrix'
+
+// @ts-ignore this is just for type checks
+declare let prisma: import('@prisma/client').PrismaClient
+
+testMatrix.setupTestSuite(
+  () => {
+    beforeAll(async () => {
+      await prisma.user.create({
```

Comment:
`createMany`?

Reply:
It don't work for SQLite so easier just to do this!